modular_skyrat/modules/bluespace_miner/code/bluespace_miner.dm

```diff
@@ -148,10 +148,10 @@
 	)
 	needs_anchored = TRUE
 
-/datum/supply_pack/misc/bluespace_miner
+/datum/supply_pack/engineering/bluespace_miner
```

Comment:
adding it to their console is a bit much, especially with how often they'd be able to press it to make ten-thousand bucks appears=

Reply:
Alright while gas miners do give 10k to cargo, Making it free isn't justifiable with a 20 minute cooldown.